scm/driver/gitlab/content.go

```diff
@@ -110,14 +110,13 @@ type updateContentBody struct {
 }
 
 type entry struct {
-	Id string `json:"id"`
+	Id   string `json:"id"`
```

Comment:
interesting is there not formatter which runs as part of the build? should that not have failed earlier?